src/toniarts/openkeeper/game/trigger/TriggerActionData.java

```diff
@@ -28,7 +28,6 @@
 public class TriggerActionData extends TriggerData {
 
     private TriggerAction.ActionType actionType;
-    private static final Logger logger = Logger.getLogger(TriggerActionData.class.getName());
```

Comment:
If you decide to remove the logger, you should probably remove the import as well